tests/test_config.py

```diff
@@ -12,6 +12,7 @@
 # limitations under the License.
 
 import json
+
```

Comment:
Can we revert the changes in this file as these are not needed in this PR.

Reply:
Reverted.